bluebox/cli.py

```diff
@@ -0,0 +1,19 @@
+"""cli.py
+
+This file can be used to interactively generate tone sequences.
+"""
+
```

Comment:
Interface should be compatible with previous version but maybe require an `-i/--interactive` flag, otherwise e.g. read stdin/pipe/file for the sequence 
Can make both space and P pause.
Allow setting device on cli, as well as SR, number of channels etc

Reply:
Also, make backends / MF registry? so if someone subclasses the base classes they can also select it via the cli?

E.g.

```python
#my_cli.py
from bluebox import cli, freq
from my_mf import MyMF

if __name__ == "__main__":
  freq.register_mf(MyMF)
  bluebox()
```